pennylane/ops/qubit/matrix_ops.py

```diff
@@ -332,3 +334,197 @@ def _controlled(self, control):
 
     def label(self, decimals=None, base_label=None, cache=None):
         return super().label(decimals=decimals, base_label=base_label or "U", cache=cache)
+
+
+class BlockEncode(Operation):
+    r"""BlockEncode(a, wires)
+    Construct a unitary :math:`U(A)` such that an arbitrary matrix :math:`A`
+    is encoded in the top-left block.
+
+    .. note::
+        If the norm of :math:`A`  is greater than 1, we normalize it to ensure
+        :math:`U(A)` is unitary. The normalization constant can be
+        accessed through :code:`op.hyperparameters["norm"]`.
+
+        Specifically, the norm is computed as the maximum of
+        :math:`L_{\infty} \vert AA^\dagger \vert` and
+        :math:`L_{\infty} \vert A^\dagger A \vert`. Where :math:`L_{\infty}`
```

Comment:
This should be just ||AA^dagger||, which is the standard notation for operator norm (common name for infinity norm)

Reply:
Change made and specified "operator norm" in the first sentence.